srcalc/src/main/java/gov/va/med/srcalc/web/view/admin/EditRiskModel.java

```diff
@@ -97,25 +276,46 @@ public int getMaxDisplayNameLength()
         return DisplayNameConditions.DISPLAY_NAME_MAX;
     }
     
+    public List<ModelTermSummary> getTermSummaries() 
```

Comment:
All of the getters in this class should have a javadoc attached. getConstantTerm() needs one too.